Source/Fuse.iOS/Fuse.iOS.unoproj

```diff
@@ -2,9 +2,6 @@
   "Copyright": "Copyright (c) Fuse Open 2018",
   "Description": "Native UI and bootstrapping for iOS",
   "Publisher": "Fuse Open",
-  "Package": {
-    "ProjectUrl": "https://fuseopen.com"
-  },
```

Comment:
This is already like this on master, perhaps you could rebase on master to make this go away? :)

Reply:
strange where that came from cos I originally rebased, anyhoo, sorted it out